storage/src/tests/visiting/visitormanagertest.cpp

```diff
@@ -36,7 +36,8 @@ namespace storage {
 namespace {
 
 using msg_ptr_vector = std::vector<api::StorageMessage::SP>;
-
+vespalib::string _Storage;
```

Comment:
Shouldn't this be initialised to _"storage"_?

Reply:
Fixed